### PR TITLE
Feature/environments config

### PR DESCRIPTION
**Pull request by liviuignat**
Changes:
- Config files for different environments
- Run mongo on CI directly

### CODE REVIEW

File: conf/application.prod.conf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Secret key
+# Will be replaced in the future
+application.secret="WILL_BE_REPLACED"
```
Comment (tudorraul) [inline conf/application.prod.conf:3]: variable name not self explanatory

File: conf/application.test.conf [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# ReactiveMongo
+mongodb.uri = "mongodb://localhost:27017/scalaplaymongo"
+
+# Only used in tests
+mongodb.dbName = "scalaplaymongo"
+
+# Override does not work, any ideas are welcome
+mongo-async-driver {
+  akka {
+    loglevel = ERROR
```
Comment (liviuignat) [inline conf/application.test.conf:19]: I have no idea why, but this line is not overriding the application.config like the rest of the configuration.

File: conf/application.prod.conf [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# Logger provided to your application:
+logger.application=DEBUG
+
+# ReactiveMongo
+mongodb.uri = ${?MONGOLAB_URI}
+
+# Override does not work, any ideas are welcome
+mongo-async-driver {
+  akka {
+    loglevel = WARN
```
Comment (liviuignat) [inline conf/application.prod.conf:23]: This one is also not overriding. Any ideas?

Comment (tudorraul) [inline conf/application.prod.conf:23]: so, can you see info level logs from the mongo driver's actors?

Comment (liviuignat) [inline conf/application.prod.conf:23]: Yes. In application.conf it is configured to be "loglevel = DEBUG" and it stays like that also for application.prod and application.test.conf. Mainly I've seen it while running tests and only by changing it in application.conf, then the debug messages are not show in the tests output.

Comment (liviuignat) [inline conf/application.prod.conf:23]: It only happens for akka config, not for the rest.

**Pull request merged**